webui-src/app/mail/mail_util.js

```diff
@@ -77,10 +90,9 @@ const MessageSummary = () => {
           ),
           m('td', files.length),
           m('td', details.title),
-          // m('td', details.rspeerid_srcId == 0 ?
-          //  '[Notification]' :
-          //  peopleUtils.getInfo(details.rspeerid_srcId)), // getInfo previously uses "/rsIdentity/getIdentitiesInfo"
-
+          m('td', details.rsgxsid_srcId === '0'.repeat(32) ?
```

Comment:
Better to cast to a number and then compare:
```suggestion
          m('td', Number(details.rsgxsid_srcId) === 0 ?
```